builder/test/pipeline.cc

```diff
@@ -906,6 +906,10 @@ TEST_P(stencil_chain, pipeline) {
     }
   }
 
+  if (split == 1 && max_workers == loop::serial) {
```

Comment:
If we think the replica pipeline generator already has good coverage, another way to get this would be to just add names to a pipeline that already has replica pipeline tests